### PR TITLE
[ticket/13789] Upgrade reCaptcha plugin to 2.0 API.

### PR DESCRIPTION
**Pull request by prototech**
[PHPBB3-13789](https://tracker.phpbb.com/browse/PHPBB3-13789)

### CODE REVIEW

**Comment by marc1706** [comment]
The lock file is not up-to-date. Please run this command to update the lock file for the recaptcha entry:
```php ../composer.phar update google/recaptcha```

**Comment by Nicofuma** [comment]
@prototech you forgot to commit the .lock file ;)

**Comment by prototech** [comment]
Updated.

File: phpBB/composer.lock [hunk 1]
```diff
@@ -1977,12 +2022,12 @@
             "version": "v1.0.2",
             "source": {
                 "type": "git",
-                "url": "https://github.com/silexphp/Pimple.git",
```
Comment (marc1706) [inline phpBB/composer.lock]: @Nicofuma why are these changes still happening?

Comment (Nicofuma) [inline phpBB/composer.lock]: I really don't know... especially that this change shouldn't happen in this direction...

**Pull request merged**